### PR TITLE
UI and template expansion for the wildcard domain

### PR DESCRIPTION
**Pull request by stephdl**
UI to create a checkbox for a domain wildcard

NethServer/dev#5390

### CODE REVIEW

File: createlinks [hunk 1]
```diff
@@ -24,6 +24,7 @@ use esmith::Build::CreateLinks  qw(:all);
 
 my @templates = (qw(
                 /etc/hosts
+                /etc/dnsmasq.conf
```
Comment (DavidePrincipi) [inline createlinks:27]: If we expand this template we invert the dependency between nethserver-hosts and nethserver-dnsmasq RPMs. As nethserver-dnsmasq is today an ISO package we can do it. However, before doing it, could you analyse the consequences of inverting the dependency direction?

**Review by DavidePrincipi — CHANGES_REQUESTED** [review]

File: createlinks (hunk 1 above)
Comment (stephdl) [reply createlinks:27]: Ok just tested it is good for me

If you add `Requires: nethserver-dnsmasq` to the spec file of nethserver-hosts you have an error, therefore you must also remove the `Requires: nethserver-hosts` of the nethserver-dnsmasq.spec

Can we go for this ?

Comment (stephdl) [reply createlinks:27]: after tested it, I'm sure it worth the game to inverse the dependency
Actually (nethserver-dnsmasq requires nethserver-hosts)
if I remove nethserver-dnsmasq, only nethserver-dnsmasq is removed
if I remove nethserver-hosts, both nethserver-hosts and nethserver-dnsmasq are removed

if I inverse the dependency (nethserver-hosts requires nethserver-dnsmasq)
if I remove nethserver-dnsmasq, both nethserver-hosts and nethserver-dnsmasq are removed
if I remove nethserver-hosts, both nethserver-hosts and nethserver-dnsmasq are removed

**Review by DavidePrincipi — APPROVED** [review]
Before merging this pr let me configure this repo for travis-ci integration: @nethbot will push RPMs for us :)

**Pull request merged**